phpstan.neon

```diff
@@ -351,6 +353,7 @@ parameters:
             message: '#Array destruct is not allowed\. Use value object to pass data instead#'
             paths:
                 - packages/BetterPhpDocParser/PhpDocParser/StaticDoctrineAnnotationParser/ArrayParser.php
+                - rules/Php74/Rector/Property/TypedPropertyRector.php
```

Comment:
This should be resolved via value object

Reply:
implemented https://github.com/rectorphp/rector-src/pull/295/commits/4faef9f05097893673582b1c566884d912d6c402